ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -703,6 +700,6 @@ def get_as_list(dict_object, key):
 
         propagator = propagators.get_global_httptextformat()
         ctx = propagator.extract(get_as_list, carrier)
-        otel_context = get_span_from_context(ctx).get_context()
+        otel_context = trace_api.get_current_span(ctx).get_context()
```

Comment:
`None` check here?

Reply:
yeah looks like it needs one. wonder why a test didn't detect this...

Reply:
fixed w/ test added.